app/komponenter/SamtykkePanel.tsx

```diff
@@ -0,0 +1,41 @@
+import { ConfirmationPanel } from '@navikt/ds-react';
+import { useState } from 'react';
+import { Button } from '@navikt/ds-react';
+import { SanityDokument } from '~/typer/sanity/sanity';
+import TekstBlokk from './tekstBlokk/tekstBlokk';
+
+interface Props {
+  innhold: SanityDokument;
+  samtykke: SanityDokument;
+  feilmelding: SanityDokument;
+}
+
+const SamtykkePanel: React.FC<Props> = ({
+  innhold,
+  samtykke,
+  feilmelding,
+}: Props) => {
+  const [samtykkeErBekreftet, settSamtykkeErBekreftet] = useState(false);
+  const [trykkVidere, settTrykkVidere] = useState(false);
+
+  return (
+    <div>
+      <ConfirmationPanel
+        checked={samtykkeErBekreftet}
+        label={<TekstBlokk tekstblokk={samtykke} />}
+        onChange={() => settSamtykkeErBekreftet(harLest => !harLest)}
```

Comment:
Hvorfor heter staten `settSamtykkeErBekreftet` men callback funksjonen tar inn `harLest`. Den burde hete en av to: 
- samtykkeErBekreftet -> da har den samme navn som staten
- prevState -> da blir det implisitt at den henviser til samtykkeErBekreftet